drizzlepac/pars/acs_header_hap.rules

```diff
@@ -401,13 +401,30 @@ MTFLAG    MTFLAG    first
 # Header Keyword Rules for remaining keywords
 #
 ################################################################################
+#
+# Keywords which need to be deleted for MVM products
+# These are not relevant for exposures from multiple proposals
+#
+################################################################################
+<delete>  IPPPSSOO
+<delete>  ASN_ID
+<delete>  ASN_MTYP
+<delete>  ASN_TAB
+<delete>  GYROMODE
+<delete>  SUNANGLE
+<delete>  MOONANGLE
```

Comment:
Is this a typo?  MOONANGL vs MOONANGLE?

Reply:
Fixed in this file along with 'acs_mvm_header_hap.rules' in a625671.